appveyor.yml

```diff
@@ -108,7 +108,7 @@ deploy:
       <summary>Interface Informations</summary>
 
       * [Strawberry Perl](http://strawberryperl.com/) 5.28
-      * ~~[ActiveTcl](http://www.activestate.com/activetcl/downloads) 8.6.6~~ (Disabled for now. See [#194](https://github.com/vim/vim-win32-installer/issues/194).)
+      * ~~[ActiveTcl](http://www.activestate.com/activetcl/downloads) 8.6.6~~ (Disabled for 32bits now. See [#194](https://github.com/vim/vim-win32-installer/issues/194).)
```

Comment:
The same change is needed in README.md.

Reply:
thanks amended the commit (and squashed)